quantized_mesh_encoder/ecef.py

```diff
@@ -16,22 +16,22 @@ def to_ecef(positions):
 
     Args:
         - positions: expected to be an ndarray with shape (-1, 3)
-    from latitude-longitude-height to
+        - ellisoid: a dict that defines an ellipsoid with with "a" and "b" values. Defaults to WGS84
+    from latitude-longitude-height to ecef
```

Comment:
I think I meant to have `from latitude-longitude-height to ecef` attached to the `positions` arg, though it could probably be worded better.